test/rsrc/convert_stub.py

```diff
@@ -20,8 +20,8 @@ def convert(in_file, out_file, tag):
     # On Windows, use Unicode paths. (The test harness gives them to us
     # as UTF-8 bytes.)
     if platform.system() == 'Windows':
-        in_file = in_file.decode('utf8')
-        out_file = out_file.decode('utf8')
+        in_file = in_file.encode('latin-1').decode('utf-8')
+        out_file = out_file.encode('latin-1').decode('utf-8')
```

Comment:
so obviously we don't wanna do latin-1 here, but the equivalent of `arg_encoding`.  It does break py3 though.